terraform/github/repositories.tf

```diff
@@ -395,3 +395,8 @@ module "modernisation-platform-incident-response" {
     "helm"
   ]
 }
+
+resource "github_repository" "modernisation-platform-terraform-ecs" {
```

Comment:
This block of code will add a new repository and archive it. However, such resource already exists and needs updating. Do a search in code for the repo name and you will find it.